### DISCUSSION TITLE
Caesar requests

### DISCUSSION THREAD

**Discussion by pavel-klavik** [Keriew/augustus#249] [Ideas]
In the later stages when you receive a request, you want to send it immediately if all the resources are available. I would add a direct button there to the message.

**Comment by cdsmith-umn** [comment]
Even better would be if you could set a configuration item to automatically dispatch requests once they can be fulfilled.

**Comment by arkimedes1967** [comment]
> Even better would be if you could set a configuration item to automatically dispatch requests once they can be fulfilled.

I do not agree with the 'automatically dispatch' because the player must **allways** have the last decision about it.

**Comment by cdsmith-umn** [comment]
It's a configuration item. As you've pointed out about those, if you don't want to use it, then don't. I don't know how you handle Caesar's requests, but if I have the resources, then the request is dispatched immediately after I receive it. A configuration item would obviate the requirement that I interrupt everything else I am doing in order to send it.

**Comment by cdsmith-umn** [comment]
Personally, I get tired of all the dialogs that I see that interrupt what I'm doing. I've got festival announcements, population announcements, Caesar's requests, gladiator revolts, barbarian attacks, working hippodromes, working colosseums, the Trade Advisor hassling me because I've opened sea routes before constructing docks, the occasional fire or building collapse, earthquakes, and the list goes on. I'd like to get down to the point where I get notified of only gladiator revolts, attacks, fires, and building collapses.

I would love it if I could schedule festivals to automatically happen and simply rotate through the gods, rather than having a dialog box and then have to go access the Entertainment Advisor to schedule the next one. Even if I could only schedule up to five festivals at a time, that would be better than what we have now.

**Comment by arkimedes1967** [comment]
> I would love it if I could schedule festivals to automatically happen and simply rotate through the gods, rather than having a dialog box and then have to go access the Entertainment Advisor to schedule the next one. Even if I could only schedule up to five festivals at a time, that would be better than what we have now.

This is my #35 request since 30th april and for Keriew _are worth considering, but are low priority for now._

I would be careful with too much automation because at some point the game will run itself and the player is somehow expendable. Festivals are recurrent that's because i support the automation. Cesar's requests are not.

Sometimes i've enough goods to send to Caesar but not enough to consumption, so i prefer increase the production first and send it later.

**Comment by crudelios** [comment]
The festivals have been addressed with the religion update.

I agree with adding a button to the message displaying `X goods stored at the warehouses - dispatch now`, but not the automatic dispatch of the items, even as a config option. To me the decision to send the goods always has to be explicitly made by the player.

**Comment by subatron** [comment]
> It's a configuration item. As you've pointed out about those, if you don't want to use it, then don't. I don't know how you handle Caesar's requests, but if I have the resources, then the request is dispatched immediately after I receive it. A configuration item would obviate the requirement that I interrupt everything else I am doing in order to send it.

I agree with others. Configuration Item or not, think of it this way:

Caesar wants 12 wine. You have 12 wine, but you're always just about to have a small villa and you really want to try to finish up your luxury palace block. If you send the 12 wine now, you won't get the casas and you'll have to spend more time because your houses devolve (and you potentially mess up other housing blocks). OR, you take the fact that Caesar, most the time, gives you 12+ months to "deliver the goods" and finish designing your blocks and get some additional stock in, just in case.

Plus, as others pointed out, there is such thing as too much automation. There are plenty of mobile games out there that are "idle, play the game for you" type games. We don't need a desktop version of that. Plus, if you play the game using the suggestions that GamerZakh has provided, the game wins itself.

Lastly - would you rather someone who communicates everything important but doesn't heed your "I don't want to hear this anymore", or someone who doesn't communicate at all? Considering how society is going with its shitty communication skills, I'd take the first, communicate everything.

**Comment by cdsmith-umn** [comment]
@subatron @crudelios I honestly don't really care whether my suggestion is ever implemented. I said it would be nice to have.
